src/rshim_fuse.c

```diff
@@ -714,6 +715,17 @@ static int rshim_fuse_misc_read(struct cuse_dev *cdev, int fflags,
       p += n;
       len -= n;
     }
+
+    /* Check whether NIC_FW is in locked mode. */
+    rc = bd->read_rshim(bd, RSHIM_CHANNEL, bd->regs->scratchpad1, &value, RSHIM_REG_SIZE_8B);
```

Comment:
We can follow existing coding style in this driver to not exceeding 80 characters per-line.